tests/test_validator_integration.py

```
@@ -0,0 +1,915 @@
+
+        not_forwarded = []
+        for field in training_fields:
+            qualified = f"training.{field}"
+            if qualified in self.KNOWN_NON_FORWARDED:
+                continue
+            kwarg_name = field_to_kwarg.get(field, field)
+            if kwarg_name not in forwarded:
+                not_forwarded.append(field)
+
+        # These fields are IN the config but NOT forwarded (dead config)
+        expected_missing = {"size_sup", "shuffle_types", "random_drop"}
```

Ruff (0.15.1)

[error] Local variable `expected_missing` is assigned to but never used

Remove assignment to unused variable `expected_missing`

(F841)